docs/content/changelog.md

```diff
@@ -1375,7 +1375,7 @@ Point release to fix hubic and azureblob backends.
     * curl install for rclone (Filip Bartodziej)
     * --stats now shows percentage, size, rate and ETA in condensed form (Ishuah Kariuki)
     * --exclude-if-present to exclude a directory if a file is present (Iakov Davydov)
-    * rmdirs: add --leave-root flag (lewpam)
+    * rmdirs: add --leave-root flag (lewapm)
```

Comment:
How did you notice the username was spelled wrong here?

And how did it get misspelled given that it is inserted by a script??

So many questions!

Reply:
I noticed because I was looking for similar words in my local aspell dictionary because I wanted to find words with inconsistent capitalization.

I don't know how it ended up being wrong. You tell me ;)